src/floating-focus.js

```diff
@@ -228,7 +207,13 @@ export default class FloatingFocus {
 	}
 
 	monitorElementPosition() {
-		if (!this.target || !this.previousTargetRect || Object.is(this.previousTargetRect, this.target.getBoundingClientRect())) {
+		if (!this.target || !this.previousTargetRect || this.floaterIsMoving) {
+			return;
+		}
+
+		const newRect = this.target.getBoundingClientRect();
+		const prevRect = this.previousTargetRect;
+		if(prevRect.top == newRect.top && prevRect.left == newRect.left) {
```

Comment:
`Object.is` checks if they're the same object, and by that it means they're pointing to the same reference. Which they're not in this case. We want to compare old to new coordinates.

See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is

Reply:
Ah, I just assumed this method was to check object equality. Weird naming then (:

I would still also check on width and height though! Agree?

Reply:
My idea was: The method does not imply size changes, nor do I expect the dimensions of the target to change as long as it has been focused. Or can you imagine a case where it will?

Reply:
To me it feels like we can never know what people will use this for, so it'd be better to be safe than sorry. For instance: what about an input field that changes size because a button pops up on the side of it at some point. Or, the parent of a button changes max-width or something for whatever reason, causing the button to shrink on focus.
So yeah, just something like 'guarding for whatever crazy stuff people come up with'